benefits/tests/test_redis_backend.py

```
@@ -0,0 +1,117 @@
+
+
+def _redis_available() -> bool:
+    if not REDIS_URL:
+        return False
+    try:
+        import redis
+
+        client = redis.from_url(REDIS_URL, socket_connect_timeout=2, **_ssl_kwargs())
+        client.ping()
+        return True
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)